README.md

```diff
@@ -55,10 +55,120 @@ console.log(hi); // => howdy world!
 
 # Publishing Binaries
 
-It's a good idea to publish pre-built binaries of your module if you want others to be able to install it properly on their own systems, which can very likely not have a compiler like `gcc` or `clang`. Node-pre-gyp does a lot of the heavy lifting for us (like detecting which system you are building on and deploying to s3) but you'll need a few things configured to get started.
+It's a good idea to publish pre-built binaries of your module if you want others to be able to easily install it on their system without needing to install a compiler like g++ or clang++. Node-pre-gyp does a lot of the heavy lifting for us (like detecting which system you are building on and deploying to s3) but you'll need a few things configured to get started.
+
+1) In the `package.json`, update the `"binary"` field to the appropriate s3 bucket `host`.
+
+For Mapbox staff we recommend using a host setting of `"host": "https://mapbox-node-binary.s3.amazonaws.com",` which will publish to `s3://mapbox-node-binary/<your module name>`.
+
+Note: for namespaced modules the path will end up being `s3://mapbox-node-binary/@org/<your module name>`.
+
+2) Copy the ci.template.js
+
+Copy the `ci.template.js` from this repo into your repo and place it at  `./cloudformation/ci.template.js`
+
+3) Install deps
+
+```
+npm install @mapbox/cfn-config
+npm install @mapbox/cloudfriend
+```
+
+4) Create a user with permissions to upload to `s3://mapbox-node-binary/<your module name>/`
+
+First configure your AWS creds. You will need to set at least the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` keys. And also `AWS_SESSION_TOKEN` if you are using 2-factor auth.
+
+Then run:
+
+```bash
+./node_modules/.bin/validate-template cloudformation/ci.template.js
```

Comment:
How about recommending a global install and then removing the `./node_modules/.bin/` so that it actually works? The `npm install -g @mapbox/cloudfriend` could be moved to `Step 4` to reduce the chance it is missed.